db_scripts/mosip_idpbinding/ddl/idpbinding-public_key_registry.sql

```diff
@@ -16,20 +16,23 @@
 -- object: public_key_registry.public_key_registry | type: TABLE --
 -- DROP TABLE IF EXISTS public_key_registry.public_key_registry CASCADE;
 CREATE TABLE public_key_registry(
+    id_hash character varying(50) NOT NULL,
 	psu_token character varying(256) NOT NULL,
 	public_key character varying(50) NOT NULL,
```

Comment:
Please don't mention the length for public key @sowmya695 
public_key character varying NOT NULL

Reply:
fixed